packages/flutter/lib/src/widgets/two_dimensional_viewport.dart

```diff
@@ -815,6 +838,7 @@ abstract class RenderTwoDimensionalViewport extends RenderBox implements RenderA
       visitor(child);
       child = parentDataOf(child)._nextSibling;
     }
+    _keepAliveBucket.values.forEach(visitor);
   }
 
   @override
```

Comment:
Trying to remember... I think this was because ensureVisible/getOffsetToReveal is not yet implemented for RenderTwoDimensionalViewport. That is in https://github.com/flutter/flutter/pull/128812. Otherwise right now it would crash, so I think I remember putting the visible part in place to prevent a crash until it was properly implemented. I almost forgot about it, I added a note to the PR for when I get back to it.